liberty-ls/src/main/java/io/openliberty/tools/langserver/LibertyConfigFileManager.java

```diff
@@ -109,6 +114,12 @@ public static boolean isServerEnvFile(LibertyTextDocument tdi) {
      */
     public static boolean isServerEnvFile(String uri) {
         String filePath = normalizeFilePath(uri);
+        for (String configDir : customConfigDirs) {
```

Comment:
I think we should check the conditions that are straight forward first (that are part of the return statement), then if that isn't true...check the config dirs.

Also, why not strip the filename off (after checking that it is `server.env`) and check if the customConfigDirs contain a match?

Reply:
Much cleaner with that method change